src/__tests__/omit.ts

```diff
@@ -0,0 +1,51 @@
+import test from 'ava';
+import * as RD from '@cala/remote-data';
+import RemoteCollection from '../index';
+import { Item, items } from './fixtures';
+
+test('with no items loaded, #omit', t => {
+  const col = new RemoteCollection<Item>('id').omit('a');
+
+  t.deepEqual(col.view(), RD.initial, 'view is set to initial');
+});
+
+test('with some items loaded, #omit', t => {
+  const col = new RemoteCollection<Item>('id').withList(items).omit('a');
+
+  t.deepEqual(col.view(), RD.success([items[1]]), 'view is reset to initial');
+});
+
+test('with view failure, #omit', t => {
+  const col = new RemoteCollection<Item>('id').withListFailure('Oh no').omit('a');
+
+  t.deepEqual(col.view(), RD.failure(['Oh no']), 'view is reset to initial');
+});
+
+test('with no items loaded at the view key, #omit', t => {
+  const col = new RemoteCollection<Item>('id')
+    .withList(items, 'someOtherViewKey')
+    .omit('a', 'someViewKey');
+
+  t.deepEqual(col.view('someViewKey'), RD.initial, 'view is set to initial');
+  t.deepEqual(col.view('someOtherViewKey'), RD.success(items));
+});
+
+test('with some items loaded at two view keys, #omit', t => {
+  const col = new RemoteCollection<Item>('id')
+    .withList(items, 'someViewKey')
+    .withList(items, 'someOtherViewKey')
+    .omit('a', 'someViewKey');
+
+  t.deepEqual(col.view('someViewKey'), RD.success([items[1]]), 'view is set to initial');
```

Comment:
This test is testing that when you `omit` an item, that it removes it from the correct list. Similar thing as what `omit` in lodash would do:

```ts
const viewOne = [{ id: 'a' }, { id: 'b' }];
const viewTwo = [...viewOne];

omit(viewOne, r => r.id === 'a'); // [{ id: 'b' }]
viewTwo; // [{ id: 'a' }, { id: 'b' }];
```

Reply:
I find it a bit confusing that you're using that call in every spec here. I'd rather just see a spec to see if omit is working rather than it be spread across specs that are testing fetches, failures, etc.

Reply:
This file is _all_ tests for `omit`, I'm just testing different states that the `RemoteCollection` might be in.

Reply:
lmao...sorry i did not read the filename! this all makes sense then.